src/springfield/fields.py

```diff
@@ -479,6 +542,7 @@ def jsonify(self, value):
     int: IntField(),
     basestring: StringField(),
     unicode: StringField(),
+    bytes: BytesField(),
```

Comment:
If str and bytes are aliases, this means that we have two duplicate keys here.

@six8 should we be mapping bytes -> ByteField() or str -> StringField()?  I don't think we can do both.  I'd probably default to not adding bytes to this mapping for backwards compatibility but I don't know if anyone is actually using this type map feature.